src/builder/pod/mod.rs

```diff
@@ -358,11 +377,12 @@ impl PodBuilder {
                 .map(|node_affinity| Affinity {
                     node_affinity: Some(node_affinity),
                     pod_affinity: self.pod_affinity.clone(),
-                    ..Affinity::default()
+                    pod_anti_affinity: self.pod_anti_affinity.clone(),
                 })
                 .or_else(|| {
                     Some(Affinity {
                         pod_affinity: self.pod_affinity.clone(),
+                        pod_anti_affinity: self.pod_anti_affinity.clone(),
                         ..Affinity::default()
                     })
                 }),
```

Comment:
Couldnt this just be (maybe im missing something):
```
            affinity: Some(Affinity {
                node_affinity,
                pod_affinity: self.pod_affinity.clone(),
                pod_anti_affinity: self.pod_anti_affinity.clone(),
            }),
```

Reply:
Ah yes! I didn't event see that. Fixed cef7ac7723e4019af0655769e46c881aaf1a86d8

Reply:
I missed the simplied property creation, I did that here: 8922df6